envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/DynamicForwardProxyTest.kt

```diff
@@ -35,19 +40,27 @@ class DynamicForwardProxyTest {
         @JvmField
         @RegisterExtension
         val envoy = EnvoyExtension(envoyControl, config = AdsDynamicForwardProxy)
+
+        @JvmField
+        @RegisterExtension
+        val httpsService = GenericServiceExtension(HttpsEchoContainer())
+
+        @JvmStatic
+        @BeforeAll
+        fun setup() {
+            envoy.container.addDnsEntry("my.example.com", httpsService.container().ipAddress())
+        }
     }
 
     @Test
     fun `should allow to request domains with suffix com`() {
         // given
-        untilAsserted {
+        untilAsserted(wait = Duration.ofSeconds(1200)) {
```

Comment:
Why does it take so long?

Reply:
My bad, was testing setup and forgot to change